gym_sts/envs/single_combat.py

```diff
@@ -0,0 +1,54 @@
+from gym_sts.spaces.observations import Observation
+
+from .base import SlayTheSpireGymEnv
+
+
+class SingleCombatSTSEnv(SlayTheSpireGymEnv):
+    def __init__(self, *args, enemy: str = "3_Sentries", **kwargs):
+        if "value_fn" not in kwargs:
+            kwargs["value_fn"] = single_combat_value
```

Comment:
I'd recommend just putting `value_fn` as a kwarg with a default in the signature.

Reply:
Done